### PR TITLE
Use Armeria to expose prometheus instead of actuator.

### PR DESCRIPTION
**Pull request by anuraaga**
This doesn't remove the fact that actuator sets up the prometheus collector, but it's a quick start.

For #2788

### CODE REVIEW

File: zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java [hunk 1]
```diff
@@ -75,7 +77,7 @@
       if (httpCollector != null) sb.annotatedService(httpCollector);
       if (healthController != null) sb.annotatedService(healthController);
       // Redirects the prometheus scrape endpoint for backward compatibility
-      sb.service("/prometheus", new RedirectService("/actuator/prometheus"));
+      sb.service("/prometheus", new PrometheusExpositionService(prometheusRegistry));
```
Comment (codefromthecrypt) [inline zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java]: If I understand correctly, we just have now 2 implementations of /prometheus. actuator is handling the other one

Comment (anuraaga) [reply zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java]: Yeah - I think that's what #2788 meant but maybe I misunderstood :)

Comment (anuraaga) [reply zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java]: Ah think I missed latest comment - so you think we should use this service on `/actuator/prometheus` too?

Comment (codefromthecrypt) [reply zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java]: well right now we aren't yanking actuator, but when we get around to it, we should mount it. that or disable the autoconfig and mount it now.

**Comment by codefromthecrypt** [comment]
ps build broke didn't look at it. will be offline mostly next 36hrs

File: zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java (hunk 1 above)
Comment (anuraaga) [reply zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java]: Went ahead and disabled the actuator/prometheus endpoint and mount it now.

**Comment by codefromthecrypt** [comment]
great. thanks rag!

**Pull request merged**